dask/dataframe/io/tests/test_io.py

```diff
@@ -553,6 +553,29 @@ def test_from_dask_array_unknown_chunks():
     assert_eq(df, pd.DataFrame(dx.compute()), check_index=False)
 
 
+def test_from_dask_array_empty_chunks():
+    # Empty chunks at the end
+    chunksizes = (1, 2, 3, 0)
+    monotonic_index = da.from_array(np.arange(sum(chunksizes)), chunks=chunksizes)
```

Comment:
```suggestion
    monotonic_index = da.from_array(np.arange(6), chunks=chunksizes)
```

Since it's alwasy 6 elements throughout this test. :)